File: app/src/main/java/ru/marslab/marsnotes/domain/model/NoteColor.java
package ru.marslab.marsnotes.domain.model;

public enum NoteColor {
    WHITE(0xFFFFFFFF, "Белый"),
    RED(0xFFFF0000, "Красный"),
    BLUE(0xFF0000FF, "Синий"),
    GREEN(0xFF00FF00, "Зеленый"),
    VIOLET(0xFFFF00FF, "Фиолетовый"),
    SKY(0xFF00FFFF, "Голубой"),
    YELLOW(0xFFFFFF00, "Желтый"),
    GRAY(0xFF555555, "Серый");

    private final int colorId;
    private final String colorName;

    NoteColor(int colorId, String colorName) {
        this.colorId = colorId;
        this.colorName = colorName;
    }

    public int getColorId() {
        return colorId;
    }

    public String getColorName() {
        return colorName;
    }
}

File: app/src/main/java/ru/marslab/marsnotes/domain/model/NoteColor.kt
package ru.marslab.marsnotes.domain.model

enum class NoteColor(val colorId: Int, val colorName: String) {
    WHITE(0xFFFFFFFF.toInt(), "Белый"),
    RED(0xFFFF0000.toInt(), "Красный"),
    BLUE(0xFF0000FF.toInt(), "Синий"),
    GREEN(0xFF00FF00.toInt(), "Зеленый"),
    VIOLET(0xFFFF00FF.toInt(), "Фиолетовый"),
    SKY(0xFF00FFFF.toInt(), "Голубой"),
    YELLOW(0xFFFFFF00.toInt(), "Желтый"),
    GRAY(0xFF555555.toInt(), "Серый");
}
